src/WebUI/Common/KubeSummary.tsx

```diff
@@ -218,6 +213,19 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
         this._selectedItemViewMap[SelectedItemKeys.ReplicaSetKey] = (item) => this._getWorkoadPodsViewComponent(item.metadata, item.spec && item.spec.template, item.kind || "ReplicaSet");
     }
 
+    private _getFiterHeaderBar(): JSX.Element {
+        return (
+            <div>
+                <ConditionalChildren renderChildren={this.state.selectedPivotKey === workloadsPivotItemKey || this.state.selectedPivotKey == undefined}>
```

Comment:
replace 
this.state.selectedPivotKey == undefined

with
!this.state.selectedPivotKey